src/models/users.model.js

```diff
@@ -98,7 +98,7 @@ module.exports = (app) => {
       timestamps: true,
     },
   );
-  users.plugin(uniqueValidator);
+  users.plugin(uniqueValidator, { message: 'That {PATH} is already registered to a user.' });
```

Comment:
Wording seems a bit weird

Reply:
I can make it "Email already registered to user" if you want, or what did you have in mind?